grumphp.yml

```diff
@@ -37,10 +37,10 @@ grumphp:
         - .github
         - .gitlab
         - .ddev
-        - /config/
-        - /drush/
+        - /config
+        - /drush
```

Comment:
Does this impact something? I see you have removed the trailing slashes everywhere.

Reply:
Yes it generates warning. Please see https://github.com/axelerant/drupal-quality-checker/issues/41